### PR TITLE
MVN_accuracy_fix_on_avx512

### PR DESCRIPTION
**Pull request by chenhu-wang**
### Details:
 - *MVN_accuracy_fix_on_avx512*
 - *fusion conditions extension*

### Tickets:
 - *55184*
 - *55340*

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@chenhu-wang what about tests? Why the issue is not reproducible by our test coverage?

**Comment by chenhu-wang** [comment]
> @chenhu-wang what about tests? Why the issue is not reproducible by our test coverage?

@dmitry-gorokhov ,the bug is in LP case, I8 computing behavior is not consistent between CPU plugin and ngraph reference. You can see [here}(https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/reference/include/ngraph/runtime/reference/mvn.hpp)the mean, variance and eps are all treated as LP while CPU plugin treated as fp32. So tests will fail when cover I8 precision. I created separate [ticket ](https://jira.devtools.intel.com/browse/CVS-56733)to do this. Local test is done with dump model blob and compare. Thank you.

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp [hunk 1]
```diff
@@ -703,12 +703,6 @@ void MKLDNNMVNNode::initSupportedPrimitiveDescriptors() {
     setPostOps(attr, true);
 
     Precision inputPrecision = getOriginalInputPrecisionAtPort(0);
-    if (getParentEdgeAt(0)->getDims().ndims() < 3 || getParentEdgeAt(0)->getDims().ndims() > 5
```
Comment (chenhu-wang) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp:706]: @dmitry-gorokhov I think this is legacy constraints, so removed, correct me please if I am wrong.

**Review by chenhu-wang — COMMENTED** [review]
> > @chenhu-wang what about tests? Why the issue is not reproducible by our test coverage?
> 
> @dmitry-gorokhov ,the bug is in LP case, I8 computing behavior is not consistent between CPU plugin and ngraph reference. You can see [here}(https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/reference/include/ngraph/runtime/reference/mvn.hpp)the mean, variance and eps are all treated as LP while CPU plugin treated as fp32. So tests will fail when cover I8 precision. I created separate [ticket ](https://jira.devtools.intel.com/browse/CVS-56733)to do this. Local test is done with dump model blob and compare. Thank you.

@dmitry-gorokhov  Partial LP tests(I8 input) is added, because we can fix set execution precision of ngraph reference as fp32, I8 input is converted and executed with fp32 in ngraph. This can cover load int8 to vmm as float. Meanwhile output is also fp32 in ngraph, can not cover I8 output for now. will done in the ticket created.

**Comment by chenhu-wang** [comment]
@mandrono ,take a look please, fusion condition is extended for the ngraph migration perf regression issue for "brain-tumor-segmentation-0002".

**Review by mandrono — APPROVED** [review]

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**